mkdocs.yml

```diff
@@ -51,13 +51,13 @@ extra:
           link: https://github.com/Autolab
         - icon: fontawesome/brands/facebook
           link: "https://facebook.com/autolabproject"
-google_analytics:
-    - "UA-79424729-3"
-    - "auto"
+    analytics:
+        provider: google
+        property: G-PH5EYTKBNQ
 markdown_extensions:
     - admonition
     - codehilite
       #linenums: true
-strict: true
+strict: false
```

Comment:
I think I forgot to change this back to true. Could you change it to true and I will approve it?

Reply:
Done.